src/app/charts/core/templates/ark-core.yaml

```diff
@@ -270,6 +270,9 @@ spec:
             - name: CATALINA_OPTS
               {{- $oidc := (include "arkcase.oidc" $ | fromYaml) -}}
               {{- $springProfiles := (not (empty $oidc)) | ternary "externalOidc" "ldap" }}
+              {{- if (include "arkcase.conf.extraSpringProfiles" $) -}}
+                {{- $springProfiles = print $springProfiles "," (include "arkcase.conf.extraSpringProfiles" $) -}}
+              {{- end -}}
```

Comment:
Few tips:

- Be *EXTREMELY* careful when using `-}}` vs `}}` ... the dash can be murder and can break your YAML syntax. When adding code within the actual template (i.e. as opposed to the very top or the very bottom, when no text is/will be rendered) you should _*always*_ use `{{-` to open, but close with `}}` (i.e. no dash), to avoid borking the line feeds and indentation
- There's no need to evaluate the template twice ... you can store the value into a variable, and then re-use it
```
{{- $extraProfiles := (include "blablbla" $) }}
{{- if $extraProfiles }}
{{- $springProfiles = (cat $springProfiles "," $extraProfiles) }}
{{- end }}
```
- Finally, there's a template called `arkcase.tools.conf` that can help you access configurations within the `global:` stanza for the chart you're working on without having to worry about all the extra nuance of the intermediate objects. This way you can just access "chart configurations" for the current chart you're working on more simply, like so:
```
{{- $extraProfiles := (include "arkcase.tools.conf" (dict "ctx" $ "value" "extraSpringProfiles")) }}
```

This will fetch the exact same value as before, but will also take into account any nuances with regards to missing maps or values (see my comment on the template implementation), so you don't have to do it over and over everywhere.